example-playbooks/replace_node/README.md

```diff
@@ -2,40 +2,50 @@
 
 This playbook will run the replace dead node procedure.
 
-### Steps:
+## Prerequisites
 
-1. Check if the node is a seed, if it is stop, seeds need to be demoted first.
-2. Apply the ansible-scylla-node role to the new node, reusing all the parameters from the existing cluster. Skip starting the scylla-server service
-3. Start the services
-4. Clean out the replace_address_first_boot record from scylla.yaml
-5. Run `nodetool repair -pr`
+* A scylla cluster with scylla-manager and scylla-monitoring installed.
```

Comment:
Here, I think you should add a section about the inventory content before you run this playbook. Am I right?

You could say something like: "Make sure to update the inventory file as described in the Usage section below before running the playbook."

Reply:
Done